modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc

```diff
@@ -7,13 +7,21 @@
 [id="defining-the-list-of-images-to-pull"]
 = Defining the list of images to pull
 
-The {image-puller-name-short} can pre-pull most images, including scratch images such as `che-machine-exec`. However, images that mount volumes in the Dockerfile, such as `traefik`, are not supported for pre-pulling on OpenShift 3.11.
+The {image-puller-name-short} can pre-pull most images, including scratch images. 
 
 .Procedure
 
-. Gather a list of relevant container images to pull by navigating to the `pass:c,a,q[{prod-url}]/plugin-registry/v3/external_images.txt` URL.
+. Get the latest {devworkspace} and {prod-short} operator bundle images, and use `podman` and `yq` to parse the related images from the ClusterServiceVersion YAML file in the /manifests/ folder to a simple list of container images:
```

Comment:
```suggestion
* Get the latest {devworkspace} and {prod-short} operator bundle images, and use `podman` and `yq` to parse the related images from the `ClusterServiceVersion` YAML file in the `/manifests/` folder to a simple list of container images:
```

Reply:
why `*` instead of `.` ? unordered list vs. ordered list?

Reply:
If the procedure contains only one step, we use `*` instead of `1.`.